### DISCUSSION TITLE
Unity Library

### DISCUSSION THREAD

**Discussion by Devigner3D** [dlemstra/Magick.NET#1361] [Help]
Greeting, 

I search for the Magick.net (ImageMagick) for unity and din't found it, i download the main branch and import it to unity in plugins but it does not work, can you help me with this please, i just want use it to read HEIC image and convert it to jpg for mobile app.

Thank you very much.

**Comment by dlemstra** [comment]
You cannot use this library in a mobile app. The Magick.Native library does not target the following `RIDs`: `ios-arm64` and `android-arm64`. I also don't know which libraries support this.
